code/game/objects/items/trader.dm

```diff
@@ -230,8 +230,8 @@ var/global/list/alcatraz_stuff = list(
 /obj/item/clothing/head/helmet/donutgiver
 	name = "donutgiver"
 	desc = "The Donutgiver III. A twenty-five sprinkle headgear with mission-variable voice-programmed confections. It has the words SPRINKLE, JELLY, CHAOS and FAVORITE etched onto its sides."
-	icon_state = "helmet_sec"
-	item_state = "helmet"
+	icon_state = "donutgiver"
+	item_state = "helmet_donuts"
 	species_fit = list(INSECT_SHAPED)
```

Comment:
I made an insectoid variant for this when it used the default helmet sprite, I'm not sure what'll occur if a bug wears this, will it just be invisible on bugs?

Reply:
The donutgiver refuses to be worn by nonhumans, did you change that part for insectoids?

Reply:
Ah shit, nah it's still restricted, stoopid racists

Reply:
Racism is the only thing stopping the vox from keeping the donut helmets to themselves!